Description: Fix "hyphen-used-as-minus-sign" warning from lintian.
Author: Samuel Bronson <naesten@gmail.com>
Forwarded: http://thread.gmane.org/gmane.comp.lib.ncurses.bugs/4805

--- tack-1.06.orig/tack.1
+++ tack-1.06/tack.1
@@ -33,7 +33,7 @@
 .SH NAME
 \fBtack\fR - \fIterminfo\fR action checker
 .SH SYNOPSIS
-\fBtack\fR [-itV] [term]
+\fBtack\fR [\-itV] [term]
 .br
 .SH DESCRIPTION
 The \fBtack\fR program has three purposes:
